tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -122,10 +126,11 @@ Get Spawner Environment Variable Value
 Spawn Notebook
     [Documentation]  Start the notebook pod spawn and wait ${spawner_timeout} seconds (DEFAULT: 600s)
     [Arguments]  ${spawner_timeout}=600 seconds
-    Click Button  Start Server
-    Wait Until Page Contains  Starting server
-    Wait Until Element Is Visible  id:progress-bar
-    Wait Until Page Does Not Contain Element  id:progress-bar  ${spawner_timeout}
+    Click Button  Start server
+    Wait Until Page Contains  Starting server  15s
+    #Wait Until Element Is Visible  id:progress-bar
+    #Wait Until Page Does Not Contain Element  id:progress-bar  ${spawner_timeout}
```

Comment:
can these commented lines be removed or are going to be used?

Reply:
The commented lines are lines I expect will need to be reverted back once we get a more finalized version of the build